shared/src/commonMain/kotlin/com/intive/picover/shared/common/uri/Uri.kt

```diff
@@ -0,0 +1,8 @@
+package com.intive.picover.shared.common.uri
+
+import dev.gitlive.firebase.storage.File
+
+expect class Uri {
```

Comment:
It was not good idea to spread usage of `dev.gitlive.firebase.storage.File` across the app, now we have more control over it